docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+          <Grid item xs={12}>
+          
+          </Grid>
+          <Grid item xs={12}>
+            <Explorer />
+          </Grid>
+        </Grid>
+      </Container>
+    </Box>
+  )
+}
+
+const datasetConfig = {
+  dataUrl: "",
+  datasetTargetDir: ".",
+  batchSize: 32,
+  seed: 42,
+  shouldCache: true,
+  trainingSetDir: "aclImdb/train",
+  testSetDir: "aclImdb/test", 
+}
+
+const datasetInfo = {
+  rawTrainSample: 
+    {textBatch: [], labelBatch: []}
+  ,
+  vectorizedSample: {
+    sampleRaw: "placeholder sample",
+    sampleVectorized: "09 90 1 1 0 0 12",
+  },
+  vocabIntValue: 0,
+  vocabTranslation: '',
+}
+
+const textVectorizationConfig = {
+  sequenceLength: 250,
+  outputMode: 'int',
+  outputModeIndex: 0,
+  outputModeAnchorEl: null,
+  tokenChips: [],
+  translationRequest: {
+    token: "",
+  }
+}
+
+const modelConfig = {
+  embeddingDim: 16,
+  showSummary: false,
+  modelName: "text-classifier-model"
+}
+
+const modelEvaluation = {
+  loss: 0.0,
+  accuracy: 0.0,
+  history: {
+    trainingLoss: [],
+    trainingBinaryAccuracy: [],
+    validationLoss: [],
+    validationBinaryAccuracy: [],
+  }
+}
+
+const trainingConfig = {
+  epochs: 4,
+  maxFeatures: 10000,
+}
+
+const Configuration = {
+  plotUri: [],
+  datasetConfig,
+  textVectorizationConfig,
+  modelConfig,
+  trainingConfig,
+  modelEvaluation,
+  datasetInfo,
+}
+
+const defaultConfig = {
+  Configuration,
+}
+
+const Tool = () => {
+  return (
+    <DesignerApi messages={{}} defaultConfig={{...defaultConfig}}>
+      <AyxAppWrapper> 
+        <App />
+      </AyxAppWrapper>
+    </DesignerApi>
+  )
+}
+
+ReactDOM.render(
+  <Tool />,
+  document.getElementById('app')
+);
+```
+While it has an impressive line count by default, our only non-boilerplate additions are skeleton frames in our return statement and default configuration defined and detailed by the `alteryx-ui` API and docs.
+
+### Module: `./src/constants.tsx`
+
+```tsx
+export const DS_CONFIG = 'datasetConfig'
+export const MDL_CONFIG = 'modelConfig'
+export const EVAL_CONFIG = 'modelEvaluation'
+```
+
+Next, we need a directory to store our components. In addition, a subdirectory for our charting component.
+You should now have
+
+```txt
+|- src/
+  |- components/
+    |- charting/
+  |- index.html
+  |- index.tsx
+  ...
+  |- webpack.prod.js
+```
+
+### Module:  `./src/components/*`
+We can now add the following to our `components` directory.
+**Note**: We are again, but for the _frontend_ SDK, creating **reusable** components we could import or place into other tools we or others develop (_depending on a publisher's license and availability_) using your components and vice-versa!
+In other words, when we create our first file below, `config-inputs.tsx`, we are creating a component to sprinkle about different places and avoid copying, pasting, and in turn, propagating stale code causing all sorts of sneaky bugs in the near (or even scarier... **FAR** future!!) future.
```

Comment:
TW review:

We can now add the following to the `components` directory.
Note: We are again, but this time for the _front-end_ SDK, creating _reusable_ components that we can import or place into other tools we or others develop (_depending on a publisher's license and availability_) using these components and vice-versa! In other words, when we create the first file below, `config-inputs.tsx`, we are creating a component to sprinkle about different places and avoid the need to copy, paste, and in turn, propagate stale code that causes all sorts of sneaky bugs in the near (or even scarier, far) future.